src/app.md

```diff
@@ -325,8 +405,15 @@ fn main() {
 [`File`]: https://doc.rust-lang.org/std/fs/struct.File.html
 [`flate2::read::GzDecoder::new`]: https://docs.rs/flate2/*/flate2/read/struct.GzDecoder.html#method.new
```

Comment:
In my updated discussion I don't reference `GzDecoder` (since it is mentioned in the `ex-tar-decompress` example). The only example that uses `GzDecoder` now is the `ex-tar-decompress` example, so I'd need to change their link as well (which I'm happy to do). Do you want me to make this change?